chris_backend/feeds/models.py

```diff
@@ -1,18 +1,23 @@
-
+from wsgiref.validate import validator
 from django.db import models
 
 import django_filters
 from django_filters.rest_framework import FilterSet
+from django.core.validators import RegexValidator
+
+Fieldtitle = RegexValidator(r"[/]*", "allow forward slash on Feed title")
```

Comment:
Please use a separate branch for each feature. Please separate out the change from https://github.com/FNNDSC/ChRIS_ultron_backEnd/pull/437 and then [ing me for a re-review.

Reply:
I have made a new pr #448 @jennydaman. I was confused with all the git structure, I tried to revert and nothing worked and then I deleted everything and start afresh